docs/angular-testing-library/api.mdx

```diff
@@ -395,20 +395,39 @@ properties that are not defined are cleared.
 
 ```typescript
 const {rerender} = await render(Counter, {
-  componentProperties: {count: 4, name: 'Sarah'},
+  componentInputs: {count: 4, name: 'Sarah'},
 })
 
 expect(screen.getByTestId('count-value').textContent).toBe('4')
 expect(screen.getByTestId('name-value').textContent).toBe('Sarah')
 
-await rerender({componentProperties: {count: 7}})
+await rerender({componentInputs: {count: 7}})
 
-// count updated to 7
+// count is updated to 7
 expect(screen.getByTestId('count-value').textContent).toBe('7')
 // name is undefined because it's not provided in rerender
 expect(screen.getByTestId('name-value').textContent).toBeUndefined()
 ```
 
+Using `partialUpdate`, only the newly provided properties will be updated. Other
+input properties that aren't provided won't be cleared.
+
+```typescript
+const {rerender} = await render(Counter, {
+  componentInputs: {count: 4, name: 'Sarah'},
+})
+
+expect(screen.getByTestId('count-value').textContent).toBe('4')
+expect(screen.getByTestId('name-value').textContent).toBe('Sarah')
+
+await rerender({componentInputs: {count: 7}, partialUpdate: true})
+
+// count is updated to 7
+expect(screen.getByTestId('count-value').textContent).toBe('7')
+// name is rendered because of the partial update
```

Comment:
Do we mean "Name is still rendered as `Sarah` because of the partial update"?

Reply:
Correct. I'll update the description to make this more clear.
Thanks for the review!